cmd/meroxa/root/apps/run.go

```diff
@@ -68,6 +70,15 @@ func (r *Run) Flags() []builder.Flag {
 
 func (r *Run) Execute(ctx context.Context) error {
 	r.path = turbineCLI.GetPath(r.flags.Path)
+	if r.path == "." {
+		r.path, _ = filepath.Abs(r.path)
+	} else if r.path == "" {
```

Comment:
How is `.` different from now providing a path? The intent is "run the app in the current working directory" right?

Could both of these branches be combined?
```
if r.path == "." || r.path == "" {
```